142 multiple choice server feedback

ysahnpark opened the pull request:
Minor changes to IpsAnswerMan to match the message structures passed back and forth with Ips/CorrecnessEngine.
Added logging here and there.

Also added Multiple Choice Question integration demo files.

@mlippert, @seannives your reviews are appreciated.

Commit:
Changed the ipsproxy integration according to modified package name

Commit:
Tab to spaces

Commit:
MultipleChoiceQuestion demo (roundtrip from AMS for init and submission)

Commit:
Submission roundtrip working. Loading config from PAF

Commit:
MCQ demo using AMS PAF Stub

seannives commented on test/integration/mcq-int.js:
```diff
@@ -0,0 +1,73 @@
+ * If you are getting error in the Browser console saying: 
+ * "XMLHttpRequest cannot load http://localhost:8088/sequencenodes/. 
+ * Origin null is not allowed by Access-Control-Allow-Origin"
+ *
```
If you get this error what does it mean?

seannives commented on test/integration/mcq-int.js:
```diff
@@ -0,0 +1,73 @@
+ * Origin null is not allowed by Access-Control-Allow-Origin"
+ *
+ * Any issue with the server, try refreshing the Redis cache
+ *
```
Could you put the command to refresh the cache here too please?

ysahnpark replied:
Good point.

The mcq-int.js is client javascript, therefore no API to refresh cache from here.
For administrative purposes, some people exposes hidden REST API to refresh cache from remote.

We could think about that too.

Commit:
Code review changes

for now don't change the structure of the studentAnswer sent from the submitManager to the AnswerMan, instead change it in the IpsAnswerMan to what the Ips needs.

A few minor changes to get the unit tests to pass.

mlippert commented:
:+1:

Commit:
Minor changes on test to use proper message
